libsol/message.c

```diff
@@ -34,8 +49,17 @@ int process_message_body(const uint8_t *message_body,
         Instruction instruction;
         BAIL_IF(parse_instruction(&parser, &instruction));
         BAIL_IF(instruction_validate(&instruction, header));
+        PRINTF("Accounts of instruction %d\n", instruction_count);
+        for (uint8_t i = 0; i < instruction.accounts_length; ++i) {
+            PRINTF("accounts[%d] = pubkeys[%d] = %.*H\n",
```

Comment:
same as above

Reply:
Compiler will remove it on its own if no instructions are inside